### PR TITLE
Fix an outdated URL in docs/update-go-version.md

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@fujitsu.com>

**What this PR does / why we need it**:
Fix an outdated URL in docs/update-go-version.md

**Special notes for your reviewer**:
The [project-infra's uploader has been moved to robots](https://github.com/kubevirt/project-infra/pull/1466)

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/update-go-version.md [hunk 1]
```diff
@@ -53,7 +53,4 @@ In addition, [go rules for bazel](https://github.com/bazelbuild/rules_go) have t
     ```
 * Visit [Bazel's releases page](https://github.com/bazelbuild/rules_go/releases) and check whether the current Bazel version supports the new Go version.
   * If it is not supported, replace the `io_bazel_rules_go` definition with the one provided in Bazel's page.
-* Use [project-infra's uploader tool](https://github.com/kubevirt/project-infra/blob/master/plugins/cmd/uploader/README.md) to upload new dependencies to dependency mirror.
```
Comment (dhiller) [inline docs/update-go-version.md:56]: Thanks for catching this. I think this is a result of changing the default branch to main.

@fossedihelm you are currently working on this also, did you already have a PR on this which we can relate to?

**Review by dhiller — APPROVED** [review]

File: docs/update-go-version.md (hunk 1 above)
Comment (fossedihelm) [reply docs/update-go-version.md:56]: Yes, [#6814](https://github.com/kubevirt/kubevirt/pull/6814) already address this issue. Thanks.

**Comment by dhiller** [comment]
@hs0210 unfortunately we have a merge conflict here, while also another PR got in that might have fixed the same issue. Could you have a look, please? Thanks!

**Comment by hs0210** [comment]
@dhiller Thanks. Yes. #6814 has fixed this, I will close this PR.

**Pull request closed without merging**